docs/developer_guide/compute.md

```diff
@@ -250,7 +250,7 @@ import starsCount from './starsCount'
       showToast(string`The repos have ${starsCount} stars`, 5000, 'success')
     ],
     catch: new Map([
-      [HttpProviderError, showToast(string`Error: ${props`error.message`}`, 5000)]
+      [HttpProviderError, showToast(string`Error: ${props`error.body.message`}`, 5000)]
```

Comment:
@henri didn't work for me when running it (from the BIN)
what i did:
i changed the repo to a name that doen't exist... then i got: Error: undefined
after changing it to .body.error it was fine...
maybe a different version of http-provider on webpackbin?